### ISSUE TITLE
NVDA + IE11 + Bootstrap Modals : NVDA moves the virtual cursor out of modal after arrow key is pressed on modal close button

### ISSUE THREAD

**Issue by ZachPeters** [nvaccess/nvda#6098]
On IE11, NVDA moves the virtual cursor out of the modal after you open a modal, tab to the close button, and hit an arrow key (up or down).
NOTE: On Chrome and Firefox this issue does not occur, and you can arrow key through the text of the modal after placing focus on the close button inside the modal.

Steps to reproduce:
1. Open the Bootstrap Modals example on the bootstrap website here: http://getbootstrap.com/javascript/#live-demo
2. Click on the "Launch demo modal" button.
3. Press tab once so that the focus is placed on the close button in the top right of the modal.
4. Hit the up or down arrow key to attempt to move the virtual cursor else where inside the modal.
5. Notice the virtual cursor is then placed outside of the modal and back onto the "Launch demo modal" button.

**Comment by Adriani90** [comment]
@ZachPeters I cannot reproduce this issue in last NVDA Alpha version anymore. Are you still having this issue?

**Comment by Adriani90** [comment]
No updates since years, I am closing as works for me. Please cooment on this if you are still having this problem and we can reopen. Thanks.
